android-plugin/src/main/java/io/ecocode/java/checks/environment/sobriety/BrightnessOverrideRule.java

```diff
@@ -49,6 +49,26 @@ public void visitNode(Tree tree) {
         if (isBrightnessAssignment(assignmentTree)) {
             Tree.Kind assignmentTreeKind = assignmentTree.expression().kind();
             switch (assignmentTreeKind) {
+                case MEMBER_SELECT:
+                    MemberSelectExpressionTree mset = (MemberSelectExpressionTree) assignmentTree.expression();
```

Comment:
OK for the code, just may be add in comment a description of the case we want to detect in this "case" which is not so trivial.